### ISSUE TITLE
bug: Artio pressure field has no units

### ISSUE THREAD

**Issue by Provider10** [yt-project/yt#2481]
### Bug report

**Bug summary**

The units under `HVAR_PRESSURE` are blank and consequently do not assign units to any pressure related derived field

**Code for reproduction**

```python
import yt as yt
ds = yt.load("~/sizmbhloz-clref04SNth-rs9_a0.9011/sizmbhloz-clref04SNth-rs9_a0.9011.art")
ds.index
ds.field_info['gas','temperature']
ds.field_info['gas','density']
ds.field_info['gas','pressure']
```

**Actual outcome**

![Screen Shot 2020-02-28 at 11 28 51 AM](https://user-images.githubusercontent.com/7698003/76636594-b7302a00-651f-11ea-8f0a-4b038a6dd094.png)

**Expected outcome**

A visualization of pressure with units that match with our system. In CGS, expected dyne/cm**2

**Version Information**

  * Operating System: macOS Catalina
  * Python Version: 3.7.3
  * yt version: 3.6.dev0
  * Other Libraries (if applicable): NONE

Installed via conda from conda-forge channel

Note: working in progress, but wanted something to refer to while I make a pull request

**Comment by matthewturk** [comment]
Hi @Provider10 !  In some other frontends, specifically chombo, we have "code pressure" defined as:

```
code_mass / (code_length * code_time**2)
```

Does that match what you think ARTIO should utilize for converting the on-disk to cgs?

**Comment by Provider10** [comment]
@matthewturk yup that's what I'm thinking. If that's what it should be, it should be an easy fix, but since I'm new to this I want to make sure I take my time and use this as a learning experience and such. I'll be a bit delayed since recent events have made research take a back seat over the past couple of days. 

The thing that's baffling me a bit is that there is a comment in fields.py that lists this field as "unused" and I don't know what it's referring to.

**Comment by matthewturk** [comment]
@Provider10 take your time!

I also noticed that and I'm not sure why.  I even did some email excavation and was unable to identify any clues as to where it came from.

**Comment by munkm** [comment]
Resolved via yt-project/yt#2521
